### PR TITLE
use new bulk upload @JSMethod

### PR DESCRIPTION
**Pull request by kevodwyer**

### CODE REVIEW

File: src/views/Drive.vue [hunk 1]
```diff
@@ -1109,12 +1112,11 @@ module.exports = {
                         //resetting .value tricks browser into allowing subsequent upload of same file(s)
                         document.getElementById('uploadFileInput').value = "";
                         document.getElementById('uploadDirectoriesInput').value = "";
-                        let future = peergos.shared.util.Futures.incomplete();
                         let progressBars = [];
                         for(var i=0; i < files.length; i++) {
                             let that = this;
-                            var thumbnailAllocation = Math.min(100000, files[i].size / 10);
-                            var resultingSize = files[i].size + thumbnailAllocation;
+                            //var thumbnailAllocation = Math.min(100000, files[i].size / 10);
```
Comment (ianopolous) [inline src/views/Drive.vue:1118]: is the thumbnailAllocation still meant to be commented?

File: src/views/Drive.vue [hunk 2]
```diff
@@ -1211,57 +1265,51 @@ module.exports = {
...
+                            component: ProgressBar,
+                            props:  {
+                            title: progress.title,
+                            done: progress.done,
+                            max: progress.max
+                            },
+                        }
                    });
-
+                //                const thumbnailAllocation = Math.min(100000, file.size / 10);
```
Comment (ianopolous) [inline src/views/Drive.vue:1283]: same question here

File: src/views/Drive.vue (hunk 1 above)
Comment (kevodwyer) [reply src/views/Drive.vue]: The thumbnailAllocation code was a bit of a hack. Previously we would fill the progress bar to 100% when the upload had finished (post thumbnail creation). No longer get a notification when individual upload finished. I have now removed the code.

File: src/views/Drive.vue (hunk 2 above)
Comment (kevodwyer) [reply src/views/Drive.vue]: code now removed

**Review by ianopolous — APPROVED** [review]
Excellent!

**Pull request merged**